source/components/RemoteConnectPage.js

```diff
@@ -85,8 +85,8 @@ class RemoteConnectPage extends Component {
 
     renderWebDAV() {
         const base = { autoCapitalize: "none", keyboardType: "default", spellCheck: false };
-        const urlInputOptions = { ...base, keyboardType: "url" };
-        const usernameInputOptions = { ...base, keyboardType: "email-address" };
+        const urlInputOptions = { ...base, keyboardType: "url", autoCorrect: false };
+        const usernameInputOptions = { ...base, keyboardType: "email-address", autoCorrect: false };
```

Comment:
Sorry bud, added these before you submitted to the `base` constant. Would you mind removing them?